tests/integration/test_opensearch.py

```diff
@@ -465,7 +465,7 @@ def test_route_through_edge(self):
         cluster_id = f"domain-{short_uid()}"
         cluster_url = f"http://localhost:{config.EDGE_PORT}/{cluster_id}"
         arn = f"arn:aws:es:us-east-1:000000000000:domain/{cluster_id}"
-        cluster = EdgeProxiedOpensearchCluster(cluster_url, arn)
+        cluster = EdgeProxiedOpensearchCluster(cluster_url, arn, cluster_url)
 
```

Comment:
This test was incorrect as far as I could tell. The cluster_url didn't adhere to the default strategy (=domain), or any strategy for that matter, so it is basically a custom endpoint, and I treated it as such with this change (because the EdgeProxiedOpensearchCluster needs this now).

Reply:
True, it effectively just tests the custom endpoint, but I wouldn't say it was wrong (because the test name states that it should test the routing through the edge, which technically it does).
But nevertheless, it's fine to add a param here if the API changes. However, you might have to change this a bit more (see the comments on the review concerning set but disabled cluster endpoints).

Reply:
:+1: